### PR TITLE
#163389679 Added authentication for comments, upvote, downvote, delete meetup, rsvp, and some validation

### PR DESCRIPTION
**Pull request by a-braham**
### What does this PR do?
Adds authentication for comments, upvote, downvote, delete meetup, rsvp, and some validation
### Description of Task to be completed?
Added authentication for meetups, comments, upvote, downvote, delete meetup, rsvp, and some validation
`/api/v2/meetups/`
`/api/v2/meetups<meetup_id>/delete`
`/api/v2/meetups<meetup_id>/rsvps`
`/api/v2/questions`
`/api/v2/question/<question_id/upvote`
`/api/v2/question/<question_id/downvote`
`/api/v2/question/<question_id/comment`

### How should this be manually tested?
- git clone this repo
`git clone https://github.com/a-braham/Questioner.git`
- Create a virtual enviroment
`virtualenv venv`
- switch into the develop branch
`git checkout develop`
- Install the requirements by running
`pip install -r requirements.txt`
- Test on postman

### What are the relevant pivotal tracker stories?
#163062185
###Screenshot
screenshot from 2019-01-23 07-23-16
screenshot from 2019-01-23 07-23-56
screenshot from 2019-01-23 07-24-37
screenshot from 2019-01-23 07-25-04
screenshot from 2019-01-23 07-25-32
screenshot from 2019-01-23 07-26-22

### CODE REVIEW

**Comment by jonathanmusila** [comment]
Nice job

File: app/api/v2/views/question_views.py [hunk 1]
```diff
@@ -140,13 +147,10 @@ def downvote(func, question_id):
...
-    # user = login_req()
-    # if not user:
-    #     return make_response(jsonify({
-    #         "status": 400,
-    #         "message": "Authentication failed: Wrong username"
-    #     })), 400
-    question = questions.oneQuestion(q_id)
+    question = questions.oneQuestion(q_id)[0]
+    user = users.login(user)[2]
+    print(user)
```
Comment (migot01) [inline app/api/v2/views/question_views.py]: remove these print statements

Comment (migot01) [reply app/api/v2/views/question_views.py]: also fix failing tests

Comment (a-braham) [reply app/api/v2/views/question_views.py]: Thank you..Will fix on those issues

**Pull request merged**